_plugins/hub.rb

```diff
@@ -29,23 +29,34 @@ module Hub
   class Generator < ::Jekyll::Generator
     safe true
 
+    # jekyll_get pulls data from team_api, but it's nested
+    def unnest_site_api_data(site)
+      site.data.keys.each do |data_type|
+        if data_type.end_with? '_raw'
+          site.data[data_type[0..-5]] = site.data[data_type]['results']
+          site.data.delete data_type
+        end
+      end
+    end
+
     # Executes all of the data processing and artifact/page generation phases
     # for the Hub.
     def generate(site)
+      unnest_site_api_data(site)
+      site.data['guest_users'] = []
+
       Joiner.join_data(site)
       Snippets.publish(site)
-      CrossReferencer.build_xrefs(site.data)
-      Canonicalizer.canonicalize_data(site.data)
       Auth.generate_artifacts(site)
-      ::HashJoiner.prune_empty_properties(site.data)
+      #::HashJoiner.prune_empty_properties(site.data)
```

Comment:
Should we think about trying to remove all of this?  I feel like I'm looking at the [Team API](https://github.com/18F/team_api gem) here and it seems odd that the work would have to be repeated, especially as we're pulling from the API itself now.

Perhaps this is something to happen in the future or maybe there is still a need I'm not aware of, but I wanted to at least raise the question.